tcms/static/style/patternfly_override.css

```diff
@@ -30,3 +30,7 @@ nav.navbar-default { border-top-color: #9ab451; }
     border: 1px solid #ccc;
     padding: 5px;
 }
+
+.c3-tooltip {
+    text-align: left;
+}
```

Comment:
This can probably go into `breakdown.css`. If necessary we will move it out later into `telemetry.css` or just rename `breakdown.css`